Sources/RxKeyboard/RxKeyboard.swift

```diff
@@ -50,17 +50,10 @@ public class RxKeyboard: NSObject, RxKeyboardType {
   // MARK: Initializing
 
   override init() {
-    #if swift(>=4.2)
-      let keyboardWillChangeFrame = UIResponder.keyboardWillChangeFrameNotification
-      let keyboardWillHide = UIResponder.keyboardWillHideNotification
-      let keyboardFrameEndKey = UIResponder.keyboardFrameEndUserInfoKey
-      let applicationDidFinishLaunching = UIApplication.didFinishLaunchingNotification
-    #else
```

Comment:
Is this change intentional? Why should we drop support for versions lower than 4.2?

Reply:
In .podspec and package.swift files swift 5.0 is listed as minimal supported version, isn't it?